qiskit/primitives/estimator_result.py

```diff
@@ -31,8 +31,11 @@ class EstimatorResult:
 
         result = estimator(circuits, observables, params)
 
-    where the i-th elements of `result` correspond to the expectation using the circuit and
-    observable given by `circuits[i]`, `observables[i]`, and the parameters bounds by `params[i]`.
+    where the i-th element of ``result.values`` corresponds to the expectation value using the circuit
```

Comment:
I was intentionally vague here with `i-th elements of result` rather than `result.values[i]` because there might be several i-th elements eg `result.values`, `result.variances`, `result.confidence_interval`, `result.metadata` etc

Reply:
I got it. I then revert them.

Reply:
I updated the sentence in a similar way to https://github.com/Qiskit/qiskit-terra/pull/7753#discussion_r822699428